plugins/CoreHome/Controller.php

```diff
@@ -300,4 +300,14 @@ public function saveViewDataTableParameters()
 
         ViewDataTableManager::saveViewDataTableParameters($login, $reportId, $parameters);
     }
+
+    public function getDateString()
+    {
+        $date = Common::getRequestVar('date', null, 'string');
+        $period = Common::getRequestVar('period', null, 'string');
+
+        $longPrettyString = $this->getLongPrettyDate($date, $period);
```

Comment:
The `getLongPrettyDate()` method doesn't take parameters, so no need to pass these two in.

Reply:
🤦‍♂️